public/index.html

```diff
@@ -9,7 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!-- <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" /> -->
```

Comment:
Do I need to change the logo for the apple bookmark icon?

Reply:
That logo file was not included in the project, so I commented the link out. It was throwing a runtime error trying to load it. To see runtime errors in Chrome, open Developer Tools and view the "Console" tab.